rest_framework/schemas/__init__.py

```diff
@@ -32,7 +32,7 @@ def get_schema_view(
         public=False, patterns=None, generator_class=None,
         authentication_classes=api_settings.DEFAULT_AUTHENTICATION_CLASSES,
         permission_classes=api_settings.DEFAULT_PERMISSION_CLASSES,
-        version=None):
+        version=None, **kwargs):
```

Comment:
I think that this would be helpfull for a more generalized approach BUT also I have specialized a get_schema_view for do that so, probably, this would sound quite useless for DRF